lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -141,6 +142,15 @@ def clone_repo(scm, remote, target, ref=nil, branch=nil, flags = nil)
     fail "Unfortunately #{scm} is not supported yet"
   end
   result = system("#{scm} #{args.flatten.join ' '}")
+  unless dirpath.nil?
+    tmpdir = SecureRandom.hex
```

Comment:
Please use http://ruby-doc.org/stdlib-2.0.0/libdoc/tmpdir/rdoc/Dir.html#method-c-mktmpdir as a secure way to generate a random temp directory.